### DISCUSSION TITLE
CustomizationProvider styles / flex

### DISCUSSION THREAD

**Discussion by espipj** [twilio-labs/paste#3118] [Q&A] [answered]
Currently CustomizationProvider doesn't accept any `styles` prop so when using it within a Flex container it's impossible to make it grow to flex 1.

**Comment by nkrantz** [comment]
Hi @espipj -

This is currently a known limitation of the Theme and Customization Providers. We have a ticket in our backlog to expose style props so that you'll be able to apply styles to the root div created by the provider. [Feel free to follow along with the ticket here](https://issues.corp.twilio.com/browse/DSYS-3855).

Thanks for providing your feedback to help improve Paste!

**Comment by TheSisb** [comment]
Hi @espipj,

The ThemeProvider div that you are trying to style has a hardcoded className called `.paste-theme-provider`.  Since you seem to be styling things outside of Paste, would you be able to target that className in your CSS and apply the styles that way?

**Reply by TheSisb** [reply, marked as answer]
Alternatively, we have now enabled styling the element via inline styles: https://github.com/twilio-labs/paste/pull/3739
Will be released in the next Paste update.

**Reply by TheSisb** [reply]
Released in `@twilio-paste/core` version `20.8.0`
